### ISSUE TITLE
Vite 5 Support

### ISSUE THREAD

**Issue by driesvints** [laravel/vite-plugin#265]
We're working on Vite 5 support which is most likely to arrive somewhere next week.

**Comment by Unnoen** [comment]
Just copying this comment from https://github.com/laravel/framework/pull/49049#issuecomment-1819174930 by @gabrielrbarbosa for a simple fix.

> I solved this by just editing vite.config.js, I think it may be easier to just add:
> 
> ```
> export default defineConfig({
>   build: {
>     manifest: 'manifest.json',
>   },
> });
> ```

The above is working for me with no noticeable issues.

**Comment by taylorotwell** [comment]
We will get to this after Pulse is released this week.

**Comment by craigrileyuk** [comment]
@Unnoen 

This works pretty well, but I'm getting a `The "code" argument must be of type number. Received type string ('SIGINT')` error when I exit dev mode, so it's not a perfect workaround.

**Comment by AntonOfTheWoods** [comment]
I agree it works "pretty well". Is there a list of things that need work?

**Comment by xsjcTony** [comment]
Any progress on this? `Vitest` released `v1` and it requires `vite v5` to work😥

**Comment by indy2kro** [comment]
The "somewhere next week" passed about 1 week ago, any idea when this could be released?

**Comment by manugch** [comment]
> The "somewhere next week" passed about 1 week ago, any idea when this could be released?

[taylorotwell](https://github.com/taylorotwell) commented [last week](https://github.com/laravel/vite-plugin/issues/265#issuecomment-1830836637)
We will get to this after Pulse is released this week.

How can you write code if you can not even read?

**Comment by gabrielrbarbosa** [comment]
@leknoppix To be honest, I've opened the first issue #253 two months ago when Vite 5.0 was in beta and it was already marked for release this quarter, also @driesvints just closed many issues as completed when people started to use stable version...

So for now I'm just using workarounds and added `--legacy-peer-deps` to `npm install` and waiting :)

**Comment by driesvints** [comment]
Hi all. We realise everyone wants Vite 5 support as soon as possible. We're doing the best we can with the resources we have and with other work also in mind. Keep an eye on our twitter to know when it's out.

**Comment by taylorotwell** [comment]
Releasing today.
